### ISSUE TITLE
[core.Command] Command invocation not recognizing triggers

### ISSUE THREAD

**Issue by ph7vc** [Arcbot-Org/Arcbot#43]
Found an issue where  the command checking method is allowing commands to happen even if they're supposed to have a trigger

**Comment by ph7vc** [comment]
Looks like it was a logic error:

```python
if(command.useDefaultTrigger and message.content.startswith(self.core.config.trigger)):
    ...
else:
    ...
```

It would skip the `useDefaultTrigger` part and just check the message anyway

**Comment by ph7vc** [comment]
Fixed in: 2a6fc8dcc05e6d6730d81fb6638d674f800e2637
